### ISSUE TITLE
prefix to name objects

### ISSUE THREAD

**Issue by mjardeli** [GoSecure/Cisco2Checkpoint#7]
Hi Martin,

If I import an existing checkpoint config ( with --cpPortsFile and/or --cpNetObjFile), if exist an exact same object name it does not check if also the port, IP address, or network is exact the same. It uses the one already existent on checkpoint database.  I have the same object name with different IP address from a previous import.

Probably to add this extra check will demand much time from you.

Is there a way to add a prefix to all object names created by your script?

thank you in advance,

**Comment by mjardeli** [comment]
Hi Martin,
I think I found my answer on config.py: 

WARN_PREFIX = '#[-] '
MSG_PREFIX = '#[+] '
NEW_NET_PREFIX = 'N_'
NEW_HOST_PREFIX = 'H_'
NEW_RANGE_PREFIX = 'R_'
NEW_GROUP_PREFIX = 'G_'
TCPUDP_PREFIX = 'TU_'
TCP_PREFIX = 'TCP_'
UDP_PREFIX = 'UDP_'

I will change it there. Maybe as a suggestion add a global parameter to add a prefix to all object names, this can be handy if importing more than one config to the same manager. It will also help identify the locality of the device you are adding.

thanks,

**Comment by martindube** [comment]
Good idea. We can keep the issue opened and mark it as a feature request.

Mart
sent from my cellphone
